global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "rollForward": "latestFeature",
+    "version": "6.0.200"
```

Comment:
Is this needed? I think in general we want to have less files sitting in the top folder if possible.

Reply:
This helps keep folks working with a more predictable build experience, and you can't put it elsewhere, so... Unfortunately I think so. This is pretty common when in other .NET projects, too, even just CLI with no IDE.

Reply:
Just occurred to me - the build failed because `dotnet format` was running with an SDK _lower than_ the actual highest target for the project. (I'll have to fix that in the PR.) If various tools happen to behave differently based on target frameworks, you can get incorrect build results. ("Worked on my machine but not the build server!") That's what this helps with, and it already caught something.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md#development-environment we need to update contributing guide to install net6.0?

Reply:
Updated `CONTRIBUTING.md` to list .NET 6.0.